gnovm/pkg/gnolang/nodes.go

```diff
@@ -2169,7 +2169,40 @@ func (x *BasicLitExpr) GetInt() int {
 	return i
 }
 
-var rePkgName = regexp.MustCompile(`^[a-z][a-z0-9_]+$`)
+var invalidPkgNames = map[string]struct{}{
+	"internal": {},
+	"crypto":   {},
+}
+
+func validatePkgPath(pkgPath, pkgName string) {
+	parts := strings.Split(pkgPath, "/")
+	lastPart := parts[len(parts)-1]
+
+	// testing env
+	if lastPart == "." {
+		return
+	}
+
+	if _, ok := invalidPkgNames[lastPart]; ok {
+		panic(fmt.Sprintf("cannot create package with invalid name %q", pkgName))
+	}
```

Comment:
For just two cases, we should use an `if` or a simple `switch`, a map is overkill and slow.

Why can `crypto` not be created?